tests/wallet/cat_wallet/test_trades.py

```diff
@@ -231,61 +231,39 @@ async def test_cat_trades(
                         "xch": {"set_remainder": True},
                         "vc": {"init": True, "set_remainder": True},
                     },
-                    post_block_balance_updates={
-                        "xch": {"set_remainder": True},
-                        "vc": {"set_remainder": True},
-                    },
+                    post_block_balance_updates={"xch": {"set_remainder": True}, "vc": {"set_remainder": True}},
                 ),
                 WalletStateTransition(
                     pre_block_balance_updates={
                         "xch": {"set_remainder": True},
                         "vc": {"init": True, "set_remainder": True},
                     },
-                    post_block_balance_updates={
-                        "xch": {"set_remainder": True},
-                        "vc": {"set_remainder": True},
-                    },
+                    post_block_balance_updates={"xch": {"set_remainder": True}, "vc": {"set_remainder": True}},
                 ),
             ]
         )
 
-        proofs_maker: VCProofs = VCProofs({"foo": "1", "bar": "1", "zap": "1"})
-        proof_root_maker: bytes32 = proofs_maker.root()
+        proofs_maker = VCProofs({"foo": "1", "bar": "1", "zap": "1"})
+        proof_root_maker = proofs_maker.root()
         await client_maker.vc_spend(
-            vc_record_maker.vc.launcher_id,
-            wallet_environments.tx_config,
-            new_proof_hash=proof_root_maker,
+            vc_record_maker.vc.launcher_id, wallet_environments.tx_config, new_proof_hash=proof_root_maker
```

Comment:
is this formatting done automatically by something?
these formatting changes dilute the relevant changes